upgrade_test.py

```diff
@@ -400,7 +400,7 @@ def _rollback_node(self, node, upgrade_sstables=True):
             else:
                 node.remoter.sudo(r'apt-get remove scylla\* -y')
                 node.remoter.sudo(
-                    rf'DEBIAN_FRONTEND=noninteractive apt-get install {node.scylla_pkg()}\* -y'
+                    rf'DEBIAN_FRONTEND=noninteractive apt-get install {node.scylla_pkg()} -y'
```

Comment:
Note to self (and other), this should change in docs

@annastuchlik @roydahan @yaronkaikov  

we shouldn't use the asterisk, I pulls packages with it when rolling back, that might break the next upgrade

Reply:
What I suspected. good.